cmd/security-file-token-provider/README.md

```diff
@@ -12,16 +12,25 @@ It should create an executable named `security-file-token-provider` in the curre
 
 ## Run
 
-Run the `integrationtest.sh` script.
+Change to the testdata directory and run the `integrationtest.sh` script.
 
 ```sh
+cd testdata
 ./integrationtest.sh
```

Comment:
You probably want to mention the prerequisite of `jq` in the documentation. I.e. jq has to be installed first before run the `./integrationtest.sh` as I was having issue on run this locally without `jq`.

Reply:
```sh
Creating edgex-secrets-setup ... done
Creating edgex-files         ... done
Creating edgex-core-consul   ... done
Creating edgex-vault         ... done
Creating edgex-vault-worker  ... done
Waiting for stack to come up...
Vault container id b6576557cf3e
./integrationtest.sh: 1: ./integrationtest.sh: jq: not found
```

Reply:
Resolved.

Reply:
Ran and validated that it is working as it intends to do